drf_model_pusher/backends.py

```diff
@@ -104,10 +117,10 @@ class PrivatePusherBackend(PusherBackend):
     class Meta:
         abstract = True
 
-    def get_channel(self, instance=None):
+    def get_channels(self, instance=None):
         """Return the channel prefixed with `private-`"""
-        channel = super().get_channel(instance=instance)
-        return "private-{channel}".format(channel=channel)
+        channel = super().get_channels(instance=instance)
+        return ["private-{channel}".format(channel=channel)]
```

Comment:
If there are multiple channels returned from `get_channels` then the return result is an array of 1 string `"private-[a, b, c]"` rather than a array `["private-a", "private-b", "private-c"]`